### PR TITLE
Added resources for Web Design

### PR DESCRIPTION
**Pull request by Shashank0271**
Added resources for web design

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1192,6 +1193,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :bulb: [SVG Backgrounds](https://www.svgbackgrounds.com/): Customize and apply backgrounds fast.
 - :bulb: [Vecteezy](https://www.vecteezy.com/): Download Free Vector Art, Stock Photos & Stock Videos.
 - :bulb: [Haikei](https://app.haikei.app/): Multiple Shape Generator for UI.
+- :bulb: [](https://freewebillustrations.com/): Discover 1,500+ Free High-Quality Web Illustrations
```
Comment (shahednasser) [inline README.md:1196]: Why is there no title to this link?

**Comment by Shashank0271** [comment]
I have made the changes

File: README.md [hunk 2]
```diff
@@ -542,6 +542,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :green_book: [Write a Flutter desktop application](https://codelabs.developers.google.com/codelabs/flutter-github-graphql-client/#0)
 - :video_camera: [Flutter Bloc Complete Course](https://www.youtube.com/watch?v=THCkkQ-V1-8)
 - :video_camera: [Flutter Bloc - from Zero To HERO](https://www.youtube.com/playlist?list=PLptHs0ZDJKt_T-oNj_6Q98v-tBnVf-S_o)
+- :video_camera: [Flutter Easy state management using Providers](https://www.youtube.com/watch?v=lR4abtROTIc&t=483s)
```
[the remark this answers is not in the document]
Comment (Shashank0271) [reply README.md]: removed it

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Thanks!

**Pull request merged**